concent-builder/repository-prepare.yml

```diff
@@ -0,0 +1,59 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - repositories.yml
+  tasks:
+    - become:      yes
+      remote_user: builder
+      block:
+        - name:    Create a directory for secrets
+          file:
+            path:  "/home/{{ shared_user }}/build/concent-secrets"
```

Comment:
Here you upload secrets to `/home/{{ shared_user }}/build` and a few steps later you delete `{{ build_dir }}`, which is the same directory. So you're deleting secrets you just uploaded.

You should upload secrets to `data_dir`.